strax/processing/peak_merging.py

```diff
@@ -85,7 +85,7 @@ def merge_peaks(peaks, start_merge_at, end_merge_at, max_buffer=int(1e5)):
 
         # Downsample the buffers into new_p['data'], new_p['data_top'],
         # and new_p['data_bot']
-        strax.store_downsampled_waveform(new_p, buffer, True, buffer_top)
+        strax.store_downsampled_waveform(new_p, buffer, True, True, 6, buffer_top)
```

Comment:
This 6 should be an argument as well. You should look for possible calls of `merge_peaks` and use `URLConfig` in these places.